.travis.yml

```
@@ -8,44 +8,51 @@ cache:
 
 matrix:
   include:
-    - python: 2.7
+    # Use python 3.8 as system executable for Poetry compatibility
+    - python: 3.8
       env:
         TOXENV=pylint
-    - python: 3.5
+        PY=2.7
+    - python: 3.8
       env:
         TOXENV=pylint
+        PY=3.5
     - python: 3.7
       env:
         TOXENV=pylint
+        PY=3.7
         DEPLOY="1"
     - python: 3.8
       env:
         TOXENV=pylint
+        PY=3.8
     - python: 3.9-dev
       env:
         TOXENV=pylint
+        PY=3.9-dev
```

Python 3.9 is stable these days.

```suggestion
        PY=3.9
```